packages/modular-scripts/react-scripts/config/webpack.config.js

```diff
@@ -53,10 +53,6 @@ const reactRefreshOverlayEntry = require.resolve(
 // makes for a smoother build process.
 const shouldInlineRuntimeChunk = process.env.INLINE_RUNTIME_CHUNK !== 'false';
 
-const imageInlineSizeLimit = parseInt(
-  process.env.IMAGE_INLINE_SIZE_LIMIT || '10000',
-);
-
```

Comment:
Limit was no longer necessary since the 'asset/resource' type is never inlined / is always outputted to a separate file. 

If we want some to be inline and others to be outputted in their own file, we can change the type 'asset' and according to the [docs](https://webpack.js.org/guides/asset-modules/#general-asset-type):

> Now webpack will automatically choose between resource and inline by following a default condition: a file with size less than 8kb will be treated as a inline module type and resource module type otherwise.

(Note: 'resource' means outputted to its own file in this context.)

Reply:
Moved to 'asset' type which mimics this functionality but with an 8kb limit instead of 10kb.